agent/src/heapstats-engines/jvmInfo.cpp

```diff
@@ -162,6 +162,7 @@ bool TJvmInfo::setHSVersion(jvmtiEnv *jvmti) {
       if (unlikely(result != 4)) {
         /*
          * Support JDK 9 EA
+         * See https://bugs.openjdk.java.net/browse/JDK-8061493
```

Comment:
This change looks good for now.
However we should change this logic after 9 GA release to adapt to [JDK-8061493](https://bugs.openjdk.java.net/browse/JDK-8061493) .